feedTheMonster.ts

```diff
@@ -292,6 +292,7 @@ class App {
   }
 
   private reinitializeSceneHandler(dataModal: DataModal): void {
+    this.titleTextElement.classList.remove(HIDE_ELEMENT_DELAY); // this is to remove hide delay class when scene resets.
```

Comment:
you can remove the class when reinitialization is called

Reply:
+1

Reply:
please check @ashwinnair-chimple